src/main/java/lotto/model/LottoMachine.java

```diff
@@ -0,0 +1,29 @@
+package lotto.model;
+
+import static lotto.LottoNumberConstants.LOTTO_NUMBER_COUNT;
+import static lotto.LottoNumberConstants.LOTTO_NUMBER_MAX;
+import static lotto.LottoNumberConstants.LOTTO_NUMBER_MIN;
+
+import java.util.HashSet;
+import java.util.Random;
+import java.util.Set;
+
+public class LottoMachine {
+
```

Comment:
유틸 객체라면 다른 개발자가 객체를 생성하지 않도록 기본생성자를 private 으로 숨겨줄 수 있겠어요 😉 

```suggestion
public class LottoMachine {
    private LottoMachine() { }
```

Reply:
유틸이라 생성자로 막아주는 것 좋아요!!
놓친 부분이었는데 캐치 감사합니다 :)